.buildkite/pipeline.yml

```diff
@@ -0,0 +1,61 @@
+steps:
```

Comment:
Since we can have multiple builds for this crate we should also test all of those builds with the pipelines.
What I am talking about is having "cargo build --feature x", "cargo build --feature y" as well as the default build. Same for cargo test. We should also check whether `cargo kcov --all` actually runs all unit tests (including the one labeled with ignored. Ideally these tests should be in a different pipeline and the default pipeline would test only "cargo test" and "cargo build" and so on.